### PR TITLE
Update documentation to match changes in CCX coach enrollment process.

### PR DESCRIPTION
**Pull request by jamiefolsom**
Very minor adjustments to the CCX documentation.

### CODE REVIEW

**Comment by jamiefolsom** [comment]
Super minor changes, but have a look, @pdpinch, to see if this looks good to you. If so, I will submit a PR to edx. Thanks!

File: en_us/shared/building_and_running_chapters/building_course/custom_courses.rst [hunk 1]
```diff
@@ -75,9 +75,11 @@ enrolled in the course.
 #. Enter the CCX coach's username or email address and select **Add CCX
    Coach**.
 
-When the CCX coach next logs into the LMS, the **CCX Coach** tab will be
-visible. The CCX coach dashboard is not accessible by course team members with
-other roles.
+The user will be added as a coach, and if they're not enrolled in the course
+yet, they will automatically be enrolled (so they can access course content).
+If they're enrolled, they will get an email to let them know. When the CCX
```
Comment (pdpinch) [inline en_us/shared/building_and_running_chapters/building_course/custom_courses.rst]: I think the coach only gets an e-mail if they aren't enrolled already, but I'm not 100% sure.

**Comment by pdpinch** [comment]
Just the one question. If you think it's right, you should just submit the PR to edX. Make sure to link to the code PR so they know they are connected.

File: en_us/shared/building_and_running_chapters/building_course/custom_courses.rst (hunk 1 above)
Comment (jamiefolsom) [inline en_us/shared/building_and_running_chapters/building_course/custom_courses.rst]: Hm good question. Will double check. And actually, I should clarify -- what you described is what I meant, but it's not clear from my language. Thanks.

**Comment by jamiefolsom** [comment]
Closing to open a PR on edx

**Pull request closed without merging**